legacy/application/models/airtime/CcFiles.php

```diff
@@ -146,7 +146,12 @@ private static function createAndImport($fileArray, $filePath, $originalFilename
 
             $importedStorageDir = Config::getStoragePath() . 'imported/' . self::getOwnerId() . '/';
             $importedDbPath = 'imported/' . self::getOwnerId() . '/';
-            $artwork = FileDataHelper::saveArtworkData($filePath, $originalFilename, $importedStorageDir, $importedDbPath);
+            $artwork = '';
+            try {
+                $artwork = FileDataHelper::saveArtworkData($filePath, $originalFilename, $importedStorageDir, $importedDbPath);
+            } catch (Exception $e) {
```

Comment:
Could we reduce the scope of this exception catching?

Reply:
this pr is a dirty one (just a reminder for later) .. currently hls pr portage on docker boorworm